OracleWebCenterContent/README.MD

```diff
@@ -0,0 +1,16 @@
+# Oracle WebCenter Content on Docker
+
+This project offers scripts to build an Oracle WebCenter Content docker image based on 12.2.1.4.0 release. Use this Docker configuration to facilitate installation, configuration, and environment setup for DevOps users. For more information about Oracle WebCenter Content, see the [Oracle WebCenter Content 12.2.1.4.0 Online Documentation](https://docs.oracle.com/en/middleware/webcenter/content/12.2.1.4/index.html).
```

Comment:
Please remove the word "docker" and rewrite the sentence as "This project offers scripts to build n Oracle WebCenter Content container image..."